### ISSUE TITLE
How to serve a pdf?

### ISSUE THREAD

**Issue by brentonmallen1** [getnikola/nikola#3157]
### Environment (if reporting a bug)

**Python Version:**
Python 3.6.5

**Nikola Version:**
Nikola v7.8.15

**Operating System:**
Mac OS 10.13.6

### Description:

I can't seem to figure out how best to serve pdf files.

Use case: I'd like to have a pdf in the `/files/` directory and serve that when called/linked in other posts/pages or serve the pdf as a page itself.

There doesn't seem to be a compiler for pdf files and I couldn't seem to find documentation on how to go about doing this (or the preferred procedure).

Any help is appreciated!

**Comment by Kwpolska** [comment]
I’m not sure what you’re asking for.

* if you want to generate a PDF for a post, there is no plugin or anything for that. You could use https://github.com/rst2pdf/rst2pdf, Pandoc and LaTeX, or some-such. Then again, users should be able to press ⌘P in their browser and get something good-looking out of it.
* if you want to serve any random PDF file, just put it in `files` in your Nikola site root and it will be copied to the output directory. (You can make any directory structure you want.)

**Comment by brentonmallen1** [comment]
I think what I was wanting to do was have a random pdf show up as a `page`.  I was able to render the pdf if it were linked in another page/post but I was wondering more if the pdf could be the page itself.

The issue I was running into when trying to do this was that I tried to add `pdf` to the list of compilers and to the `pages` tuple for source file generation (`POST and PAGES` in `conf.py`).

Upon further exploration though, it seems that an approach is to just point to the pdf in the navigation link to make it appear as it's own page.
